test/parallel/test-child-process-spawnsync-validation-errors.js

```diff
@@ -5,13 +5,17 @@ const spawnSync = require('child_process').spawnSync;
 const signals = process.binding('constants').os.signals;
 
 let invalidArgTypeError;
+let errCode = 56;
+if (common.isOpenBSD)
+  errCode = 46;
 
 if (common.isWindows) {
   invalidArgTypeError =
     common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 36);
 } else {
   invalidArgTypeError =
-    common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 56);
+    common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError },
+                        errCode);
```

Comment:
That is not an `errCode`. It is actually the amount of times this error has to be called so the test will work. So the name should be changed.

But more importantly: I am very confused that it will not be triggered the same amount of times as on other systems. Windows skips a couple ones, that is the reason why it has a different call number. But OpenBSD should be called in all cases.

Reply:
Ah, yeah my bad. I will revert this one and just leave it erroring for future investigation.

There are some other strange things happening that might be related:

in `test/sequential/test-child-process-execsync.js` I am hitting `assert(end < SLEEP);` on line 60 - `end` is consistently + ~300ms. Maybe it's a timing issue (no idea where to look / troubleshoot to find out)?